targets/extension

```diff
@@ -35,12 +41,14 @@ if [ "$XMETHOD" = 'xorg' ] || grep -q xorg /etc/crouton/targets; then
 fi
 
 # Use croutonurlhandler as a fallback URL handler
-handler='/usr/local/bin/croutonurlhandler'
-for link in x-www-browser gnome-www-browser www-browser; do
-    if ! update-alternatives --query "$link" | grep -q "$handler"; then
-        update-alternatives --install "/usr/bin/$link" "$link" "$handler" 15
-    fi
-done
+if [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
+    handler='/usr/local/bin/croutonurlhandler'
+    for link in x-www-browser gnome-www-browser www-browser; do
+        if ! update-alternatives --query "$link" | grep -q "$handler"; then
+            update-alternatives --install "/usr/bin/$link" "$link" "$handler" 15
```

Comment:
So I guess we need to add .desktop files for croutonurlhandler, which is probably a good idea anyway.  We may also want a desktop file for switching back to Chromium OS without creating a new tab (or maybe it can be the same desktop file).  Either way, I'll file a feature request; I don't think you need to include that in your PR unless you really want to :)